sdk/core/azure-core/CHANGELOG.md

```diff
@@ -19,10 +19,6 @@
 
   If the original behavior is desired, customers can adjust these timeouts by changing the `RetryDelay` and `MaxRetryDelay` fields in in the `TransportOptions` structure.
```

Comment:
Please fix the two typos:
```suggestion
  If the original behavior is desired, customers can adjust these timeouts by changing the `RetryDelay` and `MaxRetryDelay` fields in the `RetryOptions` structure.
```

FYI @LarryOsterman, those fields are in the `RetryOptions, and not the `TransportOptions`:
https://github.com/Azure/azure-sdk-for-cpp/blob/53c9da155244b72ac90ad27f515d4ea1e7505a01/sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp#L88-L100

Reply:
Thank you Ahson! Very good catch!